Kconfig

```diff
@@ -2013,6 +2013,9 @@ menu "LVGL configuration"
 			bool "vector graphic demo"
 			default n
 			depends on LV_USE_VECTOR_GRAPHIC
+		config LV_USE_DEMO_SMARTWATCH
+			bool "Smartwatch demo"
+			default n
```

Comment:
the other symbols for demos have been kept this way. I'm following the same to maintain consistency

Reply:
I understand this could be a nit, but on newly added PR the Kconfig bool symbols are being addressed to reflect the Kconfig standards, it would be good that newly added ones follow its good practices and we would enforce to do the same on the other places where this occur.

So please reconsider it.

Reply:
updated